pkg/controller/generic/transform/controller.go

```diff
@@ -240,6 +254,12 @@ func (ctrl *Controller[Input, Output]) Run(ctx context.Context, r controller.Run
 			return err
 		}
 
+		if ctrl.postTransformHook != nil {
```

Comment:
should we move it _after_ cleanupOutputs, which might call finalizer hooks?

Reply:
sure